lib/uffizzi/cli/preview.rb

```diff
@@ -251,9 +251,12 @@ def display_deployment_data(deployment, success)
     end
 
     def build_deployment_data(deployment)
+      url_server = ConfigFile.read_option(:server).strip.gsub(/\/*$/, '')
```

Comment:
The server value should already be written without spaces and extra characters. If it's not, then it should be added at the moment when we write the server, not when we read it.